File: scripts/HelloWorld.js
/*
* HelloWorld.js  -  displays the number of actually open layers 
*/

var JOptionPane = javax.swing.JOptionPane;
var Main = org.openstreetmap.josm.Main;

var numlayers = Main.getLayerManager().getLayers().size();
JOptionPane.showMessageDialog(Main.parent, "[JavaScript] Hello World! You have " + numlayers + " layer(s).");

File: scripts/HelloWorld.py
#
# HelloWorld.py  - displays the number of actually open layers
# 
from javax.swing import JOptionPane
from org.openstreetmap.josm import Main

numlayers = Main.getLayerManager().getLayers().size()	
JOptionPane.showMessageDialog(Main.parent, "[Python] Hello World! You have %s layer(s)." % numlayers)
